jetbrains-core/src/software/aws/toolkits/jetbrains/utils/ui/UiUtils.kt

```diff
@@ -21,14 +21,28 @@ import javax.swing.JComponent
 import javax.swing.JTextField
 import javax.swing.ListModel
 
-fun <T> ComboBox<T>.populateValues(selected: T? = null, updateStatus: Boolean = true, block: () -> Collection<T>) {
+/**
+ * @param defaultSelected The default selected item
```

Comment:
nit: what if we rename this to be `default` and the boolean to be `forceSelectDefault`

Reply:
Done.